go/executor/executor.go

```diff
@@ -364,9 +365,76 @@ func generateOptFlowOptimizeCodeAndExecute(cl *ir.OptimizeStmt, submitter *pytho
 	return nil
 }
 
-func (s *pythonExecutor) ExecuteOptimize(cl *ir.OptimizeStmt) error {
-	// TODO(sneaxiy): to be implemented
-	return fmt.Errorf("ExecuteOptimize is not supported in default submitter")
+func (s *pythonExecutor) ExecuteOptimize(stmt *ir.OptimizeStmt) error {
+	db, err := database.OpenAndConnectDB(s.Session.DbConnStr)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	driver, _, err := database.ParseURL(s.Session.DbConnStr)
+	if err != nil {
+		return err
+	}
+
+	fieldTypes, err := verifier.Verify(stmt.Select, db)
+	if err != nil {
+		return err
+	}
+
+	tableColumnsStrList := make([]string, 0)
+	for name, typ := range fieldTypes {
+		isVariable := false
+		for _, v := range stmt.Variables {
+			if strings.EqualFold(name, v) {
+				isVariable = true
+				break
+			}
+		}
+
+		if !isVariable {
+			continue
+		}
+
+		typ, err = fieldType(driver, typ)
+		if err != nil {
+			return err
+		}
+		tableColumnsStrList = append(tableColumnsStrList, fmt.Sprintf("%s %s", name, typ))
```

Comment:
Are we writing the original model definition data together with the result?

Reply:
No. Pyomo does not provide an official interface to save the model. It seems that only `pickle` can save the model (please see https://stackoverflow.com/questions/50834258/how-to-save-pickle-a-model-instance-in-pyomo and https://github.com/Pyomo/pyomo/blob/8eaa247ae41694edd317606b44ef04824608853e/pyomo/core/tests/unit/test_pickle.py).

But I found that saving lambda expression (the lambda expressions in objective and constraint rules) would fail. I would try to save the model in the future works.

Reply:
Well, I mean are we writing the original data in the input table to the output table together with the results?

Reply:
We only write the original data of variable columns in the result table. For example, if there are 2 variables named plants and markets in the model definition, the result data has 3 columns named plants, markets and shipment, where shipment is the solved  numeric result of the model.